public/static/docs/command-reference/metrics/show.md

```diff
@@ -22,47 +22,46 @@ show those specific metric files instead. With the `-a` or`-T` options, this
 command shows the different metrics values across all Git branches or tags,
 respectively.
 
-The optional `targets` argument can contain several metric files. With the `-R`
-option, a target can even be a directory, so that DVC recursively shows all
-metric files in it.
+The optional `targets` argument can contain one or more metric files. With the
+`-R` option, some of the target can even be directories, so that DVC recursively
+shows all metric files inside.
 
-Providing a `type` (`-t` option) overwrites the full metric specification (both
+Providing a `type` (`-t` option) overrides the full metric specification (both
 `type` and `xpath` fields) defined in the
-[DVC-file](/doc/user-guide/dvc-file-format) (usually set originally with the
-`dvc metrics modify` command).
+[DVC-file](/doc/user-guide/dvc-file-format) (with `dvc metrics modify`,
+typically).
 
 If `type` (via `-t`) is not specified and only `xpath` (`-x` option) is, only
-the `xpath` field is overwritten in its DVC-file. (DVC will first try to read
+the `xpath` field from the DVC-file is overridden. (DVC will first try to read
 `type` from the DVC-file, but it can be automatically detected by the file
 extension.)
 
-> Alternatively, see `dvc metrics modify` command to learn how to apply `-t` and
-> `-x` permanently.
+> See `dvc metrics modify` to learn how to apply `-t` and `-x` permanently.
+
+When metrics change in more recent <abbr>repository</abbr> versions, an
+alternative way to display them is `dvc metrics diff`, which shows a table of
+the changes.
 
 ## Options
 
-- `-t`, `--type` - specify a type of the metric file. Accepted values are:
-  `raw`, `json`, `tsv`, `htsv`, `csv`, `hcsv`. It will be saved into the
-  corresponding DVC-file, and used by `dvc metrics show` to determine how to
-  handle displaying metrics.
+- `-t`, `--type` - specify a type of the metric file. Accepted values are: `raw`
+  (default), `json`, `tsv`, `htsv`, `csv`, `hcsv`. It will be used to determine
```

Comment:
same here - `default` is probably incorrect

Reply:
Will address together with https://github.com/iterative/dvc.org/pull/933#pullrequestreview-356907712 above.